service/amqp/pom.xml

```diff
@@ -1,129 +1,184 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <groupId>org.kaazing</groupId>
-        <artifactId>gateway</artifactId>
-        <version>develop-SNAPSHOT</version>
-        <relativePath>../../pom.xml</relativePath>
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
-    <artifactId>gateway.service.amqp</artifactId>
-    <name>Kaazing WebSocket Gateway - AMQP Service</name>
-    <description>WebSocket Gateway AMQP Service</description>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Indentation has changed. Did you add tab characters or replace tab characters with spaces? Our standard is never to include tab characters in source files (some existing files may not always respect this but it is the objective).